client/packages/invoices/src/Returns/modals/InboundReturn/useDraftInboundReturnLines.ts

```diff
@@ -25,5 +46,26 @@ export const useDraftInboundReturnLines = (stockLineIds: string[]) => {
     });
   };
 
-  return { lines: draftLines, update };
+  const saveInboundReturn = async () => {
+    const inboundReturnLines: InboundReturnLineInput[] = draftLines.map(
+      line => {
+        const { id, reasonId, numberOfPacksReturned, stockLineId, comment } =
+          line;
+
+        return { id, stockLineId, reasonId, comment, numberOfPacksReturned };
+      }
+    );
+
+    const input: InboundReturnInput = {
+      id: FnUtils.generateUUID(),
+      customerId,
+      inboundReturnLines,
+    };
+
+    // TODO: error handling here
```

Comment:
I can do a quick session tomorrow on error types Structure/Standard, and how we use them in front end (in Stocktake Reason for example), although haven't done UI where errors are on multiple pages

Reply:
That would be great please!!